### PR TITLE
make seqlogo

### PR DESCRIPTION
**Pull request by betteridiot**
Attempting a fresh PR on a fresh build

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by betteridiot** [comment]
@bioconda/core @epruesse can you accept this?

**Comment by epruesse** [comment]
see also #12791  #12845

**Comment by betteridiot** [comment]
Thank you. Forgot to add those bread crumbs. Also #12848

**Comment by betteridiot** [comment]
This is an issue I posted because none of the tests would run. The containers would fail, have some permission error, or couldn't find pytest when testing.

**Comment by betteridiot** [comment]
@epruesse your commit has passed.

File: recipes/seqlogo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - seqlogo
+
+about:
+  home: https://github.com/betteridiot/seqlogo
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```
Comment (bgruening) [inline recipes/seqlogo/meta.yaml]: This is the path to the licence file. As you have this in your github repo you don't need to put it into this PR. Please remove the file from this PR

Comment (betteridiot) [reply recipes/seqlogo/meta.yaml]: Thank you for taking care of this

**Pull request merged**